package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-collapsible",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
Omit this file, the maintainer will increment when the version is ready to bump.